client/lib/purchases/types.ts

```diff
@@ -270,9 +277,103 @@ export interface MembershipSubscriptionsSite {
 	subscriptions: MembershipSubscription[];
 }
 
-export type GetChangePaymentMethodUrlFor = ( siteSlug: string, purchase: Purchase ) => string;
-
 export interface Owner {
 	ID: string;
 	display_name: string;
 }
+export type GetChangePaymentMethodUrlFor = ( siteSlug: string, purchase: Purchase ) => string;
+export type GetManagePurchaseUrlFor = ( siteSlug: string, attachedToPurchaseId: string ) => string;
+
+export type RenderRenewsOrExpiresOn = ( args: {
+	moment: ReturnType< typeof useLocalizedMoment >;
+	purchase: Purchase;
+	siteSlug: string | undefined;
+	translate: ReturnType< typeof useTranslate >;
+	getManagePurchaseUrlFor: GetManagePurchaseUrlFor;
+} ) => string;
+
+export type RenderRenewsOrExpiresOnLabel = ( args: {
+	purchase: Purchase;
+	translate: ReturnType< typeof useTranslate >;
+} ) => string;
+
+export interface Site {
```

Comment:
Similarly, this might be the same as `SiteDetails` in the `@automattic/data-stores` package.